docs/BreakingChanges/062 WebUtility.HtmlEncode method.md

```diff
@@ -0,0 +1,31 @@
+## 62: WebUtility.HtmlEncode method
```

Comment:
{
  "Id": "62",
  "Title": "WebUtility.HtmlEncode method",
  "Details": "For applications that target the .NET Framework 4.5, characters that are outside the Basic Multilingual Plane (BMP) round-trip correctly when they are passed to the WebUtility.HtmlDecode method.",
  "Suggestion": "This change should have no effect on current applications, but to restore the original behavior, set the targetFramework attribute of the &lt;httpRuntime&gt; element to a string other than \"4.5\". You can also set the unicodeEncodingConformance and unicodeDecodingConformance attributes of the &lt;webUtility&gt; configuration element to control this behavior independently of the targeted version of the .NET Framework.",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx#asp",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Net.WebUtility.HtmlEncode(System.String)",
    "M:System.Net.WebUtility.HtmlEncode(System.String,System.IO.TextWriter)"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": true,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": "This shouldn't affect anything. An analyzer may not add much beyond ApiPort, but we could do basic parameter analysis and potentially implement a code fix\nSource analyzer status: Pri 3, Planned",
  "IsRetargeting": true,
  "ImpactScope": 3
}

**Parsed APIs:**
- [x] M:System.Net.WebUtility.HtmlEncode(System.String)
- [x] M:System.Net.WebUtility.HtmlEncode(System.String,System.IO.TextWriter)